CHANGELOG.md

```diff
@@ -30,6 +30,9 @@ All notable changes to this project will be documented in this file based on the
 * Add `process.executable` field. #209
 * Add `process.working_directory` and `process.start`. #215
 * Reintroduce `http`. #237
+  * Move `http.response.body` to `http.response.body.content`. #239
```

Comment:
I don't think this needs a changelog entry as it did happen between two releases.

Reply:
As discussed, we'll keep the information there. It's in "Added", not in "Breaking Changes". Even though http was not in the beta 1 release, it was in ECS for a long time in development. So it's useful to note this here around the "reintroducing" notes.

Reply:
Haha, I see what caused the confusion :-)

There was one line in the "Breaking changes" that was stuck right under the section title, and apart from the actual bullet point list. So I move that. This is the only change in the breaking change section, fixing the bullet list :-)

The `body` entry is much lower in the changelog